fairlearn/metrics/_base_metrics.py

```diff
@@ -257,3 +260,122 @@ def count(y_true, y_pred) -> int:
     """
     check_consistent_length(y_true, y_pred)
     return len(y_true)
+
+
+def mean_prediction(y_true, y_pred, sample_weight=None) -> float:
+    r"""Calculate the (weighted) mean prediction.
+
+    The true values are ignored, but required as an argument in order
+    to maintain a consistent interface
+
+    Parameters
+    ----------
+    y_true : array_like
+        The true labels (ignored)
+
+    y_pred : array_like
+        The predicted labels
+
+    sample_weight : array_like
+        Optional array of sample weights
+    """
+    y_p = _convert_to_ndarray_and_squeeze(y_pred)
+    s_w = np.ones(len(y_p))
+    if sample_weight is not None:
+        s_w = _convert_to_ndarray_and_squeeze(sample_weight)
+
+    return np.dot(y_p, s_w) / s_w.sum()
+
+
+def _mean_overprediction(y_true, y_pred, sample_weight=None) -> float:
```

Comment:
I had meant to delete `_mean_overprediction` and `_mean_underprediction` when I was dealing with dashboard ghosts. (We could do it in this PR or a separate PR.)

Reply:
Done! Also found some test data related to group metric set that I deleted now.